kazoo/testing/harness.py

```diff
@@ -259,3 +284,9 @@ def setUp(self):
 
     def tearDown(self):
         self.teardown_zookeeper()
+
+    @classmethod
+    def tearDownClass(cls):
+        cluster = get_global_cluster()
```

Comment:
Oof, I have been bitten too many times by globals, I haven't looked at this, I suppose it probably makes sense here in testing, but def prefer in most cases passing around the actual instance to the class itself wherever possible.

Reply:
Yes this global thing was there since forever...

`tearDownClass` is part of the `TestCase` class lifecycle, from the stdlib: https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDownClass, this helps performing the cluster cleanup between each test case

Reply:
Oh that's right, I forgot we're still using `unittest` and not completely migrated over to `pytest`...